diff --git a/python/sglang/srt/utils/common.py b/python/sglang/srt/utils/common.py
index 8560246c6..d5c86c539 100644
--- a/python/sglang/srt/utils/common.py
+++ b/python/sglang/srt/utils/common.py
@@ -346,7 +346,7 @@ def get_float_env_var(name: str, default: float = 0.0) -> float:


 def support_triton(backend: str) -> bool:
-    return backend not in ["torch_native", "intel_amx"]
+    return False


 _ENABLE_TORCH_INFERENCE_MODE = get_bool_env_var(
